spec/install/gems/standalone_spec.rb

```diff
@@ -67,21 +83,6 @@
     include_examples "common functionality"
   end
 
-  describe "with gems with native extension" do
-    before do
-      install_gemfile <<-G, :standalone => true
-        source "file://#{gem_repo1}"
-        gem "very_simple_binary"
-      G
-    end
-
-    it "generates a bundle/bundler/setup.rb with the proper paths", :rubygems => "2.4" do
```

Comment:
Instead of testing the path, can we change this test to make sure that after running `setup.rb` we can still require the extension?